src/server/controllers/participants.js

```diff
@@ -40,8 +40,29 @@ exports.delete = (req, res) => {
   }
 };
 
-// Search for an individual participant by name
-exports.searchByName = async (req, res) => {
+// Search for an individual participant by forename
+exports.searchByForeName = async (req, res) => {
+  try {
+    const { participantName } = req.body;
+    const searchResult = await participant.findAll({
+      where: {
+        forename: {
+          [Op.like]: `%${participantName}%`,
+        },
+      },
+    });
+    if (searchResult[0]) {
+      res.send({ searchResult });
+    } else {
+      res.send({ message: 'Cant find participant with this name' });
+    }
+  } catch (error) {
+    res.send({ error });
+  }
+};
+
+// Search for an individual participant by surname
+exports.searchBySurName = async (req, res) => {
```

Comment:
@mohannadhanafi Similarly here, a good refactoring for this code would be to move all of the duplicated steps into a single function.

I think the ideal design for a search endpoint is that there's one endpoint or url that you can pass different search queries (or combinations) to, and it will handle all of the combinations.  For example, if there was one search function that let you do all of these:
`/api/v2/participants/search?forename=tom`
`/api/v2/participants/search?surname=duggan`
`/api/v2/participants/search?forename=tom&surname=duggan`
`/api/v2/participants/search?date=2018-12-09&borough=manhattan`

.. that'd be the ideal search endpoint :)